src/app/components/LinkComponent/LinkComponent.module.css

```diff
@@ -6,12 +6,14 @@
   font-size: 1.25em;
   text-decoration: none;
   transition: ease-in-out background-color 170ms;
+  text-align: center;
 }
 
 .link--primary {
   background-color: var(--secondary-color);
   padding: 1em 4em;
   border-radius: 3em;
+  width: 100%;
```

Comment:
It sounds not logical if only the primary link has a width of 100%. Maybe this should be the default for every link?

Reply:
Both the primary and the secondary have a width of 100%. I will add a round link that does not have that, so I think it will make more sense :)